smssync/src/main/java/org/addhen/smssync/presentation/service/ServiceControl.java

```diff
@@ -207,13 +207,8 @@ public boolean isConnected() {
         return Utility.isConnected(mContext);
     }
 
-    public enum SchedulerInstance {
-        INSTANCE;
-
-        private Scheduler getScheduler(Context context, FileManager fileManager, Intent intent,
-                int requestCode) {
-            return new Scheduler(context, fileManager, intent, requestCode,
-                    PendingIntent.FLAG_UPDATE_CURRENT);
-        }
+    private Scheduler getScheduler(Intent intent, int requestCode) {
```

Comment:
Good to know!  Looks like both implementations have the same number of methods, though :¬p